### PR TITLE
Add example using OpenStreetMap.

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

**Comment by justinvdm** [comment]
Looks good. Will take another look once #23 is merged in and travis is happy.

**Comment by hodgestar** [comment]
@justinvdm Ready for review.

**Comment by justinvdm** [comment]
:+1:

File: examples/lib/openstreetmap.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+var App = vumigo.App;
+var EndState = vumigo.states.EndState;
+
+var LocationApp = App.extend(function(self){
+    App.call(self, 'states:start');
+
+    self.states.add('states:start', function(name){
+        // This state will get the location data from the user and store it in
+        // the contact store. We will store and return the longitude and
+        // latitude information to the user.
```
Comment (gsvr) [inline examples/lib/openstreetmap.js]: @hodgestar Sorry to comment after the merge, things went a bit quick on this.  This comment says that it stores the longitude and latitude, but I don't think that's actually implemented.  Perhaps the intention was that `self.extract_address_data` would by default also return the longitude and latitude?  Shall I open another issue to implement this?

Comment (hodgestar) [inline examples/lib/openstreetmap.js]: @gsvr No worry. Nice catch. The comment is wrong. My intention was to make the OpenStreetMap example a big simpler than the GoogleMaps one (just to show the simpler use case). I've opened #29 for this.

Comment (gsvr) [inline examples/lib/openstreetmap.js]: @hodgestar Ah, okay cool.

**Pull request merged**